packages/client/src/StreamrClient.ts

```diff
@@ -144,6 +145,24 @@ class StreamrClientBase implements Context {
         return sub
     }
 
+    async updateEncryptionKey(opts: UpdateEncryptionKeyOptions): Promise<void> {
+        if (opts.streamId === undefined) {
+            throw new Error('streamId required')
+        }
+        const streamId = toStreamID(opts.streamId)
```

Comment:
Use method `StreamIdBuilder#toStreamID` instead for path-only format support.

```suggestion
        const streamId = toStreamID(opts.streamId)
```